buildAndroid.sh

```diff
@@ -49,7 +49,7 @@ export LDFLAGS="-fPIE -pie -fexceptions -Wl,-rpath-link=${NDK}/platforms/${platf
 export LDFLAGS_CORE="-fPIE -pie -fexceptions -L${NDK}/platforms/${platform}/arch-arm/usr/lib -L${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/libs/armeabi"
 export SYSROOT=${NDK}/platforms/${platform}/arch-arm
 export CXXFLAGS="-I${NDK}/platforms/${platform}/arch-arm/usr/include -I${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/include -I${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/libs/armeabi/include --sysroot=${SYSROOT} -g -O3 $OPTIMISE"
-DEFINES="-DOPENSMILE_BUILD -DBUILD_LIBSVM -DBUILD_SVMSMO -DBUILD_RNNFAST -DBUILD_WITHOUT_EXPERIMENTAL"
+DEFINES="-DOPENSMILE_BUILD -DBUILD_LIBSVM -DBUILD_SVMSMO -DBUILD_RNN -DBUILD_WITHOUT_EXPERIMENTAL"
```

Comment:
the BUILD_RNNFAST flag seems to be from a previous version and has no effect. I added BUILD_RNN sine I use the LSTM VAD component. The flag is active in buildStandalone.sh so I thought I can add it as well in buildAndroid (the VAD now also works on android after the modifications in rnn.cpp below)